en/console-and-shells.rst

```diff
@@ -354,6 +354,19 @@ By default, a ``requested`` extra param is automatically added when a Shell
 is dispatched using ``dispatchShell()``. This ``requested`` parameter prevents
 the CakePHP console welcome message from being displayed on dispatched shells.
 
+Exit Status Codes
+-----------------
+It can sometimes be useful to return exit status codes. Simply use php's ``exit()`` statement.
```

Comment:
Further currently I did not find much about exit status codes in the docs. $this->error() stopping the app feels a bit strange.